### ISSUE TITLE
e1s fails to render a cluster's "services" view when the number of services is divisible by 10

### ISSUE THREAD

**Issue by mdb** [keidarcy/e1s#116]
Currently, `e1s` silently fails to render a cluster's "services" view when the number of services fetched via the AWS API is divisible by 10. While no error is surfaced in the `e1s` UI, the `/tmp/debug-e1s.log` reveals the following clue:

```
2024/01/30 06:54:31 e1s - aws failed to describe services, err: operation error ECS: DescribeServices, https response error StatusCode: 400, RequestID: REDACTED, InvalidParameterException: Services cannot be empty
2024/01/30 06:54:31 e1s - show services page failed, error: operation error ECS: DescribeServices, https response error StatusCode: 400, RequestID: REDACTED InvalidParameterException: Services cannot be empty
```

**Comment by keidarcy** [comment]
Hi! Thank you for using e1s and contribution!
Please upgrade to v1.0.15 or newer version to check it.

**Comment by mdb** [comment]
Thanks @keidarcy! I tested that v1.0.16 does indeed contain the fix (as well as the #118 fix). Although, it's extremely slow (> 20 seconds) when a cluster hosts a large volume of services. PR #126 helps significantly, though still takes 4-5 seconds.

**Comment by keidarcy** [comment]
Hi @mdb! I merged your PR and published v1.0.17!
